data/human/ships.txt

```diff
@@ -1260,6 +1260,61 @@ ship "Cruiser"
 
 
 
+ship "Cutlass"
+	plural "Cutlasses"
+	sprite "ship/pirate cutlass"
+	thumbnail "thumbnail/pirate cutlass"
+	attributes
+		category "Light Warship"
+		"cost" 3500000
+		"shields" 5800
+		"hull" 1800
+		"required crew" 8
+		"bunks" 12
+		"mass" 173
+		"drag" 3.
+		"heat dissipation" .75
+		"fuel capacity" 600
+		"cargo space" 56
+		"outfit space" 297
+		"weapon capacity" 141
+		"engine capacity" 90
+		weapon
+			"blast radius" 72
+			"shield damage" 720
+			"hull damage" 360
+			"hit force" 1080
+	outfits
+		"Heavy Laser" 4
+		"Heavy Laser Turret"
+		
+		"Dwarf Core"
+		"LP144a Battery Pack"
+		"D14-RN Shield Generator"
+		"Small Radar Jammer"
+		"Tactical Scanner"
+		
+		"A370 Atomic Thruster"
+		"A255 Atomic Steering"
+		"Hyperdrive"
+		
+	engine -12 57.5 0.7
+	engine 12 57.5 0.7
+	gun -16 -38 
+	gun 16 -38 
+	gun -28.5 -27.5
+	gun 28.5 -27.5
+	turret 0 -1 "Heavy Laser Turret"
+	leak "leak" 50 50
+	leak "flame" 80 80
+	explode "tiny explosion" 10
+	explode "small explosion" 20
+	explode "medium explosion" 15
+	"final explode" "final explosion small"
+	description `This ship is  a modified version of a captured Navy Gunboat. Although there is little most pirates can do to improve the already advanced shielding they can still weld on a few extra hull reinforcements and an extra pair of guns giving them a tough and reasonably fast gun platform to quickly attack, plunder, and escape.`
```

Comment:
```suggestion
	description `This ship is a modified version of a captured Navy Gunboat. Although there is little most pirates can do to improve the already advanced shielding, they can still weld on a few extra hull reinforcements and an extra pair of guns. This gives them a tough and reasonably fast gun platform to quickly attack, plunder, and escape.`
```
Extra space and comma needed, but also that sentence really would be best broken up. It could still be improved on from this, for sure, but I thought this was the simplest change.